source/containers.tex

```diff
@@ -17163,7 +17163,7 @@
                   const key_compare& comp = key_compare())
       : @\exposid{c}@(std::move(cont)), @\exposid{compare}@(comp) { }
     template<class Allocator>
-      flat_multiset(sorted_equivalent_t, const container_type&, const Allocator& a);
+      flat_multiset(sorted_equivalent_t, const container_type& cont, const Allocator& a);
```

Comment:
This agrees with the form of the redeclaration in `[flat.multiset.cons]`, as is consistent with LWG style.